resources/assets/js/mixins/status.js

```diff
@@ -23,7 +23,7 @@ export default {
                 this.status.error = 'Error '
                     + err.status
                     + ' -  '
-                    + 'Neem een print screen en neem contact op met de servicedesk.';
+                    + (err.body.message || 'Neem een print screen en neem contact op met de servicedesk.');
```

Comment:
was aggreed in meeting, that message was not usefull...

Reply:
Discussed and closed.

Reply:
*agreed